src/modules/Transcript/index.tsx

```diff
@@ -780,7 +793,22 @@ function useAudioSync(
         }
         // If we found a paragraph for the current audio, use it
         else if (audioParagraph) {
-            setCurrentParagraph(audioParagraph.id);
+            const scrollHeight = scrollContainer?.scrollHeight;
+            const offsetHeight = scrollContainer?.offsetHeight;
+            const scrollTop = scrollContainer?.scrollTop;
+            let isAtBottom = false;
+            if (scrollHeight && offsetHeight && scrollTop) {
+                // 180 gives us some buffer to decide they're close enough to the bottom
```

Comment:
This is the hackiest part I don't know why offsetHeight + scrollTop don't equal scrollHeight when it is at the bottom... sometimes the difference is 12, sometimes 88, sometimes 126, and I saw it as high as 180.. maybe it has to do w/ the partial or the chrome? it was inconsistent either way... 180 should give us enough buffer to say, if they're within 180 pixels of the bottom of the scroll container, we're going to select the partial